influxdb/src/main/scala/akka/stream/alpakka/influxdb/InfluxDBWriteMessage.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.influxdb
+
+import akka.NotUsed
+
+object InfluxDBWriteMessage {
+  // Apply method to use when not using passThrough
+  def apply[T](point: T): InfluxDBWriteMessage[T, NotUsed] =
+    InfluxDBWriteMessage(point = point, passThrough = NotUsed)
+
+  // Java-api - without passThrough
+  def create[T](point: T): InfluxDBWriteMessage[T, NotUsed] =
+    InfluxDBWriteMessage(point, NotUsed)
+
+  // Java-api - with passThrough
+  def create[T, C](point: T, passThrough: C) =
+    InfluxDBWriteMessage(point, passThrough)
+}
+
+final case class InfluxDBWriteMessage[T, C](point: T,
+                                            passThrough: C,
+                                            databaseName: Option[String] = None,
```

Comment:
A bit surprising for me to see `databaseName` per element. Isn't that a candidate to have in the settings? But I guess that is an InfluxDB specific thing.

Reply:
This functionality is pretty close to using a schema prefix when using sql. The connection string for the connectivity is not bound to the database (although you can set a default database on the driver) thus the user has the ability to specify the database per measurement.